packages/openneuro-app/src/scripts/dataset/routes/snapshot.tsx

```diff
@@ -128,6 +128,14 @@ const SnapshotRoute = ({
               You must add at least one change message to create a new version
             </small>
           )}
+          {snapshots.length === 0
+            ? (
+              <p>
+                Once a version has been created and DOI assigned, the dataset
+                can no longer be deleted.
+              </p>
```

Comment:
Does it make sense to make this a checkbox, e.g.:

* [ ] I acknowledge that by creating a snapshot and assigning a DOI, I will no longer be able to delete this dataset.